AppIntegrity/Sources/AppIntegrity/JWTGenerator.swift

```diff
@@ -0,0 +1,3 @@
+public protocol JWTGenerator {
+    func generateJWT(header: [String: String], payload: [String: Any]) -> String
```

Comment:
Think we probably want `[String: Any]` as the expected type for the header parameter 🙂 as well